apps/dashboard/src/pages/questionnaire-page.tsx

```diff
@@ -0,0 +1,14 @@
+import { PageMeta } from '@/components/page-meta';
+import { AuthCard } from '../components/auth/auth-card';
+import { QuestionnaireForm } from '../components/auth/questionnaire-form';
+
+export function QuestionnairePage() {
+  return (
+    <>
+      <PageMeta title="Setup your workspace" />
+      <AuthCard>
```

Comment:
The background image repeats vertically.
<img width="1190" alt="Screenshot 2024-11-25 at 23 25 55" src="https://github.com/user-attachments/assets/f93502bd-dd9e-455d-a58e-2cb251ec699c">